syntax = "proto3";

service Showtime{
    rpc GetShowtimeByDate(ShowtimeDate) returns (ShowtimeData) {}
    rpc GetListShowtimes(Empty) returns (stream  ShowtimeData) {}
}

message ShowtimeDate {
    string date = 1 ;
}

message ShowtimeData {
    string date = 1;
    string movies = 2;
}

message Empty {
}